python/pyspark/sql/session.py

```diff
@@ -348,6 +344,11 @@ def _repr_html_(self) -> str:
             sc_HTML=self.sparkContext._repr_html_(),  # type: ignore[attr-defined]
         )
 
+    @property
+    def _jconf(self) -> SparkConf:
+        """Accessor for the JVM SQL-specific configurations"""
+        return self._jsparkSession.sessionState().conf()
```

Comment:
I think we should get this from SQLConf.get to respect the current `SparkSession`. I will do it in a separate PR.